wp-content/themes/missilethreat/inc/template-tags.php

```diff
@@ -184,16 +183,15 @@ function missilethreat_authors_list_extended()
 		global $post;
 
 		if (function_exists('coauthors_posts_links')) {
-			$authors = '<h2 class="heading">Authors</h2>';
 
 			foreach (get_coauthors() as $coauthor) {
 				$name = $coauthor->display_name;
 
-				if ( $coauthor->website ) {
```

Comment:
It looks like `user_url` is the `WP_user` equivalent of `website` for guest authors.  Should I change it back to `website`?

https://bekarice.com/adding-co-authors-plus-support-theme/

Reply:
I would. My recommendation is that every "author" gets a Guest Author account. There's an option then to map real users to guest authors, which would allow you to use the `website` field.